book-talk-be/api/src/main/resources/application.yml

```
@@ -12,4 +12,9 @@ spring:
   config:
     import:
       - classpath:application-data.yml
-      - classpath:application-lib.yml
\ No newline at end of file
+      - classpath:application-lib.yml
+
+app:
+  cors:
+    allowed-origins:
+      - ${APP_CORS_ALLOWED_ORIGINS}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)